core/payment.go

```diff
@@ -95,3 +95,24 @@ func (p PaymentSystem) CheckToken(UUID uuid.UUID, token string) error {
 	}
 	return nil
 }
+
+func (p PaymentSystem) NewAccount(userUUID uuid.UUID, account *models.Account) error {
```

Comment:
Тут `PaymentSystem` передається копіюванням, а не через поїнтер.
```suggestion
func (p *PaymentSystem) NewAccount(userUUID uuid.UUID, account *models.Account) error {
```